backend/Controllers/RecipeController.js

```diff
@@ -182,15 +183,84 @@ const updateRecipe = async (req, res)=>{
  * @description Deletes a Recipe
  * @access private
  */
-const deleteRecipe = async (req, res)=>{
-  try{
-    await Recipe.deleteOne({_id: req.body.id});
-    return res.status(200).json({success: true})
-  }catch(error){
-   console.log(error);
-   res.status(404).json({ success: false, message: "Internal server error" });
+const deleteRecipe = async (req, res) => {
+  try {
+    const recipe = await Recipe.findById(req.body.id);
+
+    if (!recipe) {
+      return res.status(404).json({ success: false, message: "Recipe not found" });
+    }
+
+    const imageName = recipe.image;
+    const owner = 'AlfiyaSiddique';
```

Comment:
@sadath2001 one improvement:
Let's store the owner, repo and branch name in env.
Also add dummy environment variable .env file with just example values. write tho short comment why this repo and owner variable required in .env file.
After making the variable can you change the hardcoded values to env values wherever name. repo and branch is used?

Reply:
hi @AlfiyaSiddique , sure thing, i can do that.
let me make the change and get back to you